src/server.cpp

```diff
@@ -3677,11 +3686,17 @@ namespace {
 
 bool Server::dynamicAddMedia(const DynamicMediaArgs &a)
 {
-	std::string filename = a.filename;
-	std::string filepath;
+	if (!m_env && (!a.to_player.empty() || a.ephemeral)) {
+		errorstream << "Server: "
+			"adding ephemeral or player-specific media at startup is nonsense"
```

Comment:
No traceback, no hint. Please add a file name to the log message such that the culprit can be found more easily.